data/src/main/java/me/androidbox/data/alarm_manager/AlarmSchedulerImp.kt

```diff
@@ -13,30 +13,43 @@ class AlarmSchedulerImp @Inject constructor(
     @ApplicationContext private val context: Context,
```

Comment:
No need to add @ApplicationContext , if you already have it on the Hilt Module

Reply:
But if you don't provide this class in the Hilt module, you do need it here